templates/nextflow-ecs-cluster.yaml

```diff
@@ -116,6 +116,40 @@ Resources:
       LogGroupName: !Join ['-', [ECSLogGroup, !Ref 'AWS::StackName']]
       RetentionInDays: 14
 
+  EcsLaunchTemplate:
```

Comment:
This `LaunchTemplate` configuration sets the exact same parameters as the existing `LaunchConfiguration` except for [AssociatePublicIpAddress](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-autoscaling-launchconfiguration.html#cfn-autoscaling-launchconfiguration-associatepublicipaddress) which is not supported in LaunchTemplate.  Also the launch template defaults to a non-public address setting when set to a non-default subnet which is what is what is being done in the `EcsAutoScalingGroup` resource.

from the docs..
```
If the instance is launched into a nondefault subnet,
the default is not to assign a public IPv4 address
```

Reply:
Thanks for the info. The changes make sense, but should we remove the LaunchConfiguration?